test/plugins/features/movies/controller.test.js

```diff
@@ -137,4 +137,26 @@ describe('movie controller', () => {
 
   });
 
+  describe('associates a location with a movie', () => {
```

Comment:
We usually name these describe blocks with the name of the controller function. So in this case it would be `assignLocation`.